src/containers/task-management/task_detail.tsx

```diff
@@ -201,8 +203,10 @@ class TaskDetail extends React.Component<RouteComponentProps, IState> {
                               task: parentTaskId,
                             })}
                           >
-                            {Constants.TASK_NAMES[parentTask.name] ||
-                              parentTask.name}
+                            {i18n._(
+                              Constants.TASK_NAMES[parentTask.name] ||
+                                parentTask.name,
+                            )}
```

Comment:
```suggestion
                            {i18n._(
                              Constants.TASK_NAMES[parentTask.name]) ||
                                parentTask.name
                            }
```

(+- prettier anyway :))